Dockerfile.local

```diff
@@ -4,6 +4,7 @@ ENV GOCACHE=/go/.go/cache GOPATH=/go/.go/path TZ=Europe/London PATH=$PATH:$GOPAT
 
 # Install github.com/cespare/reflex
 RUN GOBIN=/bin go install github.com/cespare/reflex@v0.3.1
+RUN GOBIN=/bin go install github.com/kevinburke/go-bindata/v4/...@latest
```

Comment:
You should not be using @latest, because this could pull in bad code.
This needs to be pinned to a version number.

Reply:
This is now addressed.